### PR TITLE
Andrew footer

### PR DESCRIPTION
**Pull request by andyb2**

### CODE REVIEW

File: src/components/shared/footer/Footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  background-color: var.$lightGrey;
+  @include mixins.pagePadding;
+}
+
+.footerNav {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  max-width: 1131px;
```
Comment (orrymevorach) [inline src/components/shared/footer/Footer.module.scss:19]: This 1131px is a common value that is being used as a wrapper. Can we add it to the pagePadding mixin? That way every component that uses the wrapper will have the same max width and padding.

File: src/components/shared/footer/footer-items/footer-items.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Link from 'next/link';
+import styles from './footer-items.module.scss';
+
+export default function FooterItems() {
+  const footerNavItems = [
+    { label: 'Contact', href: '/contact' },
+    { label: 'Newsletter', href: '/newsletter' },
+    { label: 'Shipping & Policies', href: '/shipping' },
+    { label: 'Special Projects', href: 'projects' },
```
Comment (orrymevorach) [inline src/components/shared/footer/footer-items/footer-items.jsx:9]: missing a slash on this href

File: src/components/shared/footer/Footer.jsx
Comment (orrymevorach) [inline src/components/shared/footer/Footer.jsx:1]: This filename and the css one as well should be lower case. 

Also can you create index.jsx files for all of these components? I don't think we ever talked about that, but let's follow that convention it allows for cleaner imports.

File: src/components/shared/nav-bar/nav-bar.module.scss [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
-  margin: 0 auto;
   width: 100%;
   max-width: 1131px;
   height: 120px;
-  z-index: 100;
-  @include mixins.pagePadding;
-  background-color: var.$white;
+}
+
+.background {
```
Comment (orrymevorach) [inline src/components/shared/nav-bar/nav-bar.module.scss:25]: can you make this classname more specific? background of what, is it the entire nav?

File: src/components/shared/nav-bar/nav-bar.module.scss
Comment (orrymevorach) [inline src/components/shared/nav-bar/nav-bar.module.scss:1]: This is a pretty decent sized refactor, how come your work on the footer affected the work on the nav bar?

File: src/components/shared/svg/nora-logo/nora-logo.jsx [hunk 4]
```diff
@@ -1,11 +1,10 @@
 import Image from 'next/image';
 import ncLogo from 'images/NC.png';
-import styles from './nora-logo.module.scss';
 
 export default function NoraLogo() {
   return (
-    <div className={styles.container}>
+    <>
       <Image src={ncLogo} alt='main logo' />
```
Comment (orrymevorach) [inline src/components/shared/svg/nora-logo/nora-logo.jsx:7]: I appreciate this though, but this doesn't need to a component in my opinion, its just one line of code. 😂

If you prefer to leave it as a component that's cool, but its also not an svg so i would relocate it to another folder.

File: src/styles/mixins.module.scss [hunk 5]
```diff
@@ -34,6 +34,11 @@
   color: #231f20;
 }
 
+@mixin openSansRegular {
+  font-family: 'OpenSans Extra Regular';
```
Comment (orrymevorach) [inline src/styles/mixins.module.scss:38]: We already have a mixin that uses openSans, im pretty sure its just called openSans, can you use that one? Also there is not font that is called Open Sans Extra Regualr, so Im pretty sure this mixin won't work properly.

File: src/components/shared/footer/Footer.module.scss (hunk 1 above)
Comment (andyb2) [reply src/components/shared/footer/Footer.module.scss:19]: I went ahead and created variables for max-widths in the variables.module.scss in the rem-mixin pr. Theres an issue if you add a max width including padding on the same container. It makes the child elements padded smaller than it should be.

File: src/components/shared/footer/desktop-footer/desktop-footer.jsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import FooterItems from '../footer-items/footer-items';
+import SocialButtons from '../social-buttons/social-buttons';
+import Copyright from '../copyright/copyright';
+import styles from './desktop-footer.module.scss';
+import SmallLogo from '../small-logo/small-logo';
+
+export default function DesktopFooter() {
```
Comment (orrymevorach) [inline src/components/shared/footer/desktop-footer/desktop-footer.jsx:7]: clean as a whistle 👌🏻

**Review by orrymevorach — APPROVED** [review]

**Pull request merged**